splash.py

```diff
@@ -6,6 +6,11 @@
 def splash(i, o):
     image = PIL.Image.open("splash.png").convert('L')
     image = invert(image)
-    image = image.convert(o.device.mode)
```

Comment:
Splash is only supported on some screens, and there isn't any logic in place to really know what device we're running on, we can't make decision about how to proceed. And since this code is currently `device` directly, it won't work now that the emulator is out of process. 

I can fix it immediately, so that it works in the emulator (requires that first-party API) or I can just let it crash (and catch the exception) for the case of the emulator and fix it in a follow-up pull request. What do you think?